src/controllers/StudentsRESTController.ts

```diff
@@ -44,6 +44,16 @@ export class StudentsRESTController extends BaseController {
         return this.ok(Result.ok(dto));
     }
 
+    @DELETE
+    @Path(":id")
+    public async deleteStudent(@PathParam("id") id: string): Promise<Envelope> {
+        const student = await this.studentsController.getStudent(id);
+        if (!student) throw RuntimeErrors.general.recordNotFound(Student);
+
+        await this.studentsController.deleteStudent(student);
+        return this.ok(Result.ok(undefined));
```

Comment:
You shouldn't return http status 200 if there is no content. Use 204 (NoContent) instead.